resources/guides/wifi_guide.md

```diff
@@ -0,0 +1,71 @@
+---
+id: wifi_guide
+# title: Wi-Fi Guide # ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
+sidebar_label: Wi-Fi Guide # ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
+slug: /guides/wifi
+---
+# Wi-Fi Guide ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
+<!-- ![alt text](https://www.transparentpng.com/thumb/wi-fi/wifi-logo-blue-png-photo-31.png 'Wifi Logo') -->
+
+This is a guide to connecting to Wi-Fi on campus.
+
+## Wi-Fi Networks available
+When you connect your wireless Wi-Fi device (phone/PC/Mac) to the network, you will see the available wireless networks listed which you can try to connect to. Along with a few others, the primary list will include:
+- **eduroam**
+- **uwinsecure**
+- **uwindsor**
+
+Which should you use? The short answer is “Connect to eduroam”. The eduroam wireless network is available to all wireless users (Students/Faculty/Staff/guests). Plus it has the additional benefit that users from other eduroam institutions (many universities, for example) can also connect to the University of Windsor’s wireless network using their home institution’s crededentials.
+
+## Eduroam - The Preferred Wireless Network
+![alt text](https://www.uwindsor.ca/itservices/sites/uwindsor.ca.itservices/files/246studentlaw_sm-retouched_crop.jpg "girl connecting to the right choice")
+
+**Eduroam** is a network used by many universities, and your UWindsor sign in will work at any campus that uses eduroam.
+1. Open Wi-Fi settings on your device and select the “eduroam” network
```

Comment:
I think we should split it up by device or OS. for example, the instructions for linux are much different than something like windows

Reply:
+1, bump.

Reply:
ok I only know what to do for Windows so maybe someone else can add on for Linux

Reply:
Can we add a section for Iphone or android? Please add issues for platforms that you dont have. for example, an issue like `Wifi instructions for linux`

Reply:
ok sounds good